src/pages/SignIn/SignIn.tsx

```diff
@@ -40,14 +42,15 @@ const SignIn = () => {
   const onSubmit = async (payload: SignInFormInputs) => {
     try {
       await login(payload);
-      navigate('/home', { replace: true });
+      // const redirectUrl = res.isAdmin ? '/admin': '/student';
+      navigate('/admin', { replace: true }); // for now, until we get back user data from backend
     } catch (error) {
       setErrorMessages(error as ErrorMessageType[]);
     }
   };
 
   useEffect(() => {
-    getCsrfToken();
+    getCsrfToken().then(() => logout());
```

Comment:
maybe not needed or there's a better way to do it? but i was trying to make the user to be logged out whenever they go to the sign in page. We did this in my previous company, and I think it makes sense to enforce the user to logout when they access the sign in page?

let me know what you think!

Reply:
That makes sense to me!

Reply:
make sense, if a user go to sign in page when they are logged in usually he wants to login with another account.